.github/workflows/release-build.yml

```
@@ -5,12 +5,8 @@ on:
   workflow_call:
     inputs:
       publish:
-        description: "Set to the release metadata JSON to publish the release"
-        required: false
-        type: string
-      publickey:
-        description: "Minisign public key. Required when publishing"
-        required: false
+        description: "The release metadata JSON"
+        required: true
```

```suggestion
        required: false
```

Publish needs to be optional or supplied with a default value

---

Yep, got it